static_src/components/users_invite.jsx

```diff
@@ -0,0 +1,73 @@
+
```

Comment:
I think this PR should focus on the backend and we should remove these frontend components for now